streamer/proxy_node.py

```diff
@@ -55,12 +63,57 @@
   pass
 
 
+class RateLimiter(object):
+  """A rate limiter that tracks which files we have written to recently."""
+
+  def __init__(self) -> None:
+    self._reset()
+
+  def suppress(self, path) -> bool:
+    """Returns true if you should skip this upload."""
+
+    now = self._now()
+    if self._last_check != now:
+      self._reset()
```

Comment:
nit: i think `time.now() > self._last_check + RATE_LIMITER_PERIOD_IN_SECONDS` is more intuitive.

Reply:
That's fair.  An earlier draft had the timestamp as a key in a dictionary for some dumb reason, but without that, dividing by the period and rounding are not necessary.

Reply:
Done